src/lib/Zikula/Bundle/CoreInstallerBundle/Resources/views/Install/dbcreds.html.twig

```diff
@@ -51,9 +51,7 @@
             <div class="col-sm-9">
                 {{ form_errors(form.database_name) }}
                 {{ form_widget(form.database_name) }}
-                <em class="text-danger">
-                    {{ __('Ensure the database is in UTF8 format.') }}
-                </em>
+                <p class="help-block">{{ __('Ensure the database is in UTF-8 format.') }}</p>
```

Comment:
@craigh I updated the pull request. Both classes are used now, so your desired color is applied.